README.rst

```diff
@@ -24,6 +24,18 @@ a `-`), the default tagged image will be `ORG/IMG:DIST-vX-test` (instead of
 `make push` will push all the images that would be generated by `make build`
 (same rules apply for `TAG`).
 
+Docker Images Publishing
+========================
+
+Images are automatically published for tags via
+[Travis](https://travis-ci.org/sociomantic-tsunami/cachalot) to
+[DockerHub](https://travis-ci.org/sociomantic-tsunami/cachalot/settings)'s
+[sociomantictsunami](https://cloud.docker.com/u/sociomantictsunami/repository/list)
+organization.
+
+Credentials used to push the images are stored in the [Travis repository
+settings(https://travis-ci.org/sociomantic-tsunami/cachalot/settings).
```

Comment:
there is a missing `]` here. Probably this is the reason why the other links added above do not render properly.

Reply:
Good catch, @daniel-zullo-sociomantic , I will create a new PR with the fix